This file is part of MXE. See LICENSE.md for licensing information.

From f271a094f8add34df3f31ac6fb12c9fe683f763a Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sat, 10 Oct 2015 16:40:35 +0100
Subject: [PATCH] add rockspec

Source:https://luarocks.org/modules/gvvaughan/lpeg/1.0.0-1
---
 lpeg-1.0.0-1.rockspec | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 lpeg-1.0.0-1.rockspec

diff --git a/lpeg-1.0.0-1.rockspec b/lpeg-1.0.0-1.rockspec
new file mode 100644
index 0000000..b4fdd27
--- /dev/null
+++ b/lpeg-1.0.0-1.rockspec
@@ -0,0 +1,32 @@
+package = "LPeg"
+version = "1.0.0-1"
+source = {
+   url = "http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.0.tar.gz",
+   md5 = "0aec64ccd13996202ad0c099e2877ece",
+}
+description = {
+   summary = "Parsing Expression Grammars For Lua",
+   detailed = [[
+      LPeg is a new pattern-matching library for Lua, based on Parsing
+      Expression Grammars (PEGs). The nice thing about PEGs is that it
+      has a formal basis (instead of being an ad-hoc set of features),
+      allows an efficient and simple implementation, and does most things
+      we expect from a pattern-matching library (and more, as we can
+      define entire grammars).
+   ]],
+   homepage = "http://www.inf.puc-rio.br/~roberto/lpeg.html",
+   maintainer = "Gary V. Vaughan <gary@vaughan.pe>",
+   license = "MIT/X11"
+}
+dependencies = {
+   "lua >= 5.1"
+}
+build = {
+   type = "builtin",
+   modules = {
+      lpeg = {
+         "lpcap.c", "lpcode.c", "lpprint.c", "lptree.c", "lpvm.c"
+      },
+      re = "re.lua"
+   }
+}
-- 
2.1.4

